packages/flutter_localizations/lib/src/l10n/material_ko.arb

```diff
@@ -0,0 +1,34 @@
+{
+  "scriptCategory": "dense",
+  "timeOfDayFormat": "ah:mm",
```

Comment:
In our system these strings only affect the layout of the time picker, but _technically_ Korean format has one space between "a" and "h:mm", i.e. "a h:mm" (see http://demo.icu-project.org/icu-bin/locexp?d_=en&_=ko_KR)

Reply:
OK, fixed.